packages/devtools_app/lib/src/inspector/inspector_tree.dart

```diff
@@ -30,14 +30,13 @@ final RegExp assertionThrownBuildingError = RegExp(
 
 typedef TreeEventCallback = void Function(InspectorTreeNode node);
 
-const Color selectedRowBackgroundColor = ThemedColor(
-  Color.fromARGB(255, 202, 191, 69),
-  Color.fromARGB(255, 99, 101, 103),
-);
-const Color hoverColor = ThemedColor(
-  Colors.yellowAccent,
-  Color.fromARGB(255, 70, 73, 76),
-);
+extension InspectorColorScheme on ColorScheme {
```

Comment:
as a future CL we should merge some of the various color scheme extensions that have grown organically.